index.html

```diff
@@ -1,579 +1,998 @@
       ],
-      status: "CG-DRAFT",
-      publisher: "Digital Verification Community Group"
-    },
-    "DID-RESOLUTION": {
-      title: "Decentralized Identifier Resolution",
-      href: "https://w3c-ccg.github.io/did-resolution/",
+      // authors, add as many as you like.
+      // This is optional, uncomment if you have authors as well as editors.
+      // only "name" is required. Same format as editors.
       authors: [
-        "Markus Sabadello",
-        "Dmitri Zagidulin"
+        {
+          name: "Guillaume Dardelet",
+          company: "Transmute",
+          companyURL: "https://www.transmute.industries/",
+        },
       ],
-      status: "Draft Community Group Report",
-      publisher: "Credentials Community Group"
-    }
-  },
-  github: {
-          repoURL: "https://github.com/w3c/did-imp-guide/",
-          branch: "master"
-  },
-  includePermalinks: false,
-
-  // editors, add as many as you like
-  // only "name" is required
-  editors: [{
-      name: "TBD"
-  }],
-
-  // authors, add as many as you like.
-  // This is optional, uncomment if you have authors as well as editors.
-  // only "name" is required. Same format as editors.
-  authors: [{
-      name: "TBD"
-  }]
-};
+      // name of the WG
+      group: "credentials",
+
+      // name (with the @w3c.org) of the public mailing to which comments are due
+      wgPublicList: "public-credentials",
+
+      // URI of the patent status for this WG, for Rec-track documents
+      // !!!! IMPORTANT !!!!
+      // This is important for Rec-track documents, do not copy a patent URI from a random
+      // document unless you know what you're doing. If in doubt ask your friendly neighborhood
+      // Team Contact.
+      //wgPatentURI:  "https://www.w3.org/2004/01/pp-impl/98922/status",
+      maxTocLevel: 2,
+      inlineCSS: true,
+    };
   </script>
   <style>
-  pre .highlight {
-    font-weight: bold;
-    color: green;
-  }
-  pre .comment {
-    color: SteelBlue;
-    -webkit-user-select: none;
-    -moz-user-select: none;
-    -ms-user-select: none;
-    user-select: none;
-  }
+    pre .highlight {
+      font-weight: bold;
+      color: green;
+    }
+
+    pre .comment {
+      font-weight: bold;
+      color: Gray;
+    }
+
+    .color-text {
+      font-weight: bold;
+      text-shadow: -1px 0 black, 0 1px black, 1px 0 black, 0 -1px black;
+    }
+
+    .simple {
+      width: 100%;
+    }
   </style>
 </head>
+
 <body>
-  <section id='abstract'>
+  <section id="abstract">
     <p>
+      This document provides guidance to DID Method developers, related to
+      serialization, cryptographic primitive selection, governance and
+      interoperability.
     </p>
     <p>
-This document provides guidance to implementers that are developing software
-for the Decentralized Identifier ecosystem.
+      Guidance provided in this document does reflect the consenus of the W3C
+      DID WG, although some members of the WG have contributed to it.
```

Comment:
Does or does not reflect the consensus of the DID WG?

Reply:
good catch, should be does not.

Reply:
taken in https://github.com/w3c/did-imp-guide/pull/3/commits/bd0b11ad2437afeceb4f332706635a3b41141789